content/docs/guides/gke/troubleshooting-gke.md

```diff
@@ -24,6 +24,64 @@ Here are some tips for troubleshooting Cloud IAP.
 
  * Make sure you are using HTTPS
 
+### DNS name not registered
+
+This section provides troubleshooting information for problems creating a DNS entry for your ingress. This section assumes
+you are using Cloud Endpoints and the DNS name
+
+```
+https://${DEPLOYMENT_NAME}.endpoints.${PROJECT}.cloud.goog
+```
+
+Symptoms
+
+  * When you access the the URL in chrome you get the error: **server IP address could not be found**
+  * nslookup for the domain name doesn't return the IP address associated with the ingress
+
+    ```
+    nslookup ${DEPLOYMENT_NAME}.endpoints.${PROJECT}.cloud.goog
+    Server:   127.0.0.1
+    Address:  127.0.0.1#53
+
+    ** server can't find ${DEPLOYMENT_NAME}.endpoints.${PROJECT}.cloud.goog: NXDOMAIN
+    ```
+
+Troubleshooting
+
+1. Check the cloud endpoints resource
+
+   ```
+   kubectl get cloudendpoints -o yaml ${DEPLOYMENT_NAME}
+   kubectl describe cloudendpoints ${DEPLOYMENT_NAME}
+   ```
+
+   * Check if there are errors indicating problems creating the endpoint
+
+1. The status of the cloudendpoints object will contain the cloud operation used to register the operation
+
+   * e.g
+
+     ```
+      status:
+        config: ""
+        configMapHash: ""
+        configSubmit: operations/serviceConfigs.jlewi-1218-001.endpoints.cloud-ml-dev.cloud.goog:43fe6c6f-eb9c-41d0-ac85-b547fc3e6e38
+        endpoint: jlewi-1218-001.endpoints.cloud-ml-dev.cloud.goog
+        ingressIP: 35.227.243.83
+        jwtAudiences: null
+        lastAppliedSig: 4f3b903a06a683b380bf1aac1deca72792472429
+        observedGeneration: 1
+        stateCurrent: ENDPOINT_SUBMIT_PENDING
+
+     ```
+
+  * You can check the status of the operation
```

Comment:
It's best to include the colon ":" at the end of this sentence, and all similar sentences in this PR. (It's tempting to assume people will know that the code below is an explanation of how to do the thing, but that's often assuming too much when the reader is working at the edge of their comprehension of this particular topic.)